shared/toks-components/src/utils/quiz.ts

```diff
@@ -0,0 +1,51 @@
+import { QuizStatus } from '@depromeet/toks-components';
+
+export const getQuizItemStatus = (openDate: Date, limitDate: Date) => {
+  const currentDate = new Date();
+
+  if (limitDate.getTime() <= currentDate.getTime()) {
+    return 'DONE';
+  } else if (currentDate.getTime() < openDate.getTime()) {
+    return 'TO_DO';
+  } else {
+    return 'IN_PROGRESS';
+  }
+};
+
+export const getTimerByQuizStatus = (
+  currentDate: Date,
+  duration: number,
+  limitDate: Date,
+  quizItemStatus: QuizStatus
+) => {
+  if (quizItemStatus === 'DONE') {
+    return '00:00:00';
+  } else if (quizItemStatus === 'TO_DO') {
+    return convertMilliSecondToString(duration);
+  } else {
+    return calculateRemainingTimerValue(currentDate, limitDate);
+  }
```

Comment:
요거도 개선이 필요해보입니당.

key, value를 통한 로직이 더 낫다고 봐요~~

```suggestion
   
  const getTimer = {
        DONE: "00:00:00",
        TO_DO: convertMilliSecondToString(duration);
         IN_PROGRESS?: calculateRemainingTimerValue(currentDate, limitDate); // 확인필요
  }
  
  const timerValue = getTimer[getItemStatus]
```

Reply:
그리고 이거 00:00:00이 아니라 convert 해줄 함수를 만들고 숫자 값으로 관리하는게 좋아보이는데 요렇게 따로 하신 이유가 있나요?

Reply:
단순히 "DONE" 상태 즉 끝났으면 무조건 00:00:00이 나와야 해서 저렇게 해주었습니다!
추가적으로 저렇게 퀴즈 상태에 따라서 값의 반환을 나누고 있는 이유는 타이머가 돌아갈때, 안돌아갈때를 구분을 짓고 싶었고 안돌아갈때는 00:00:00 혹은 제한시간으로만 나와야 하기 때문에 또 분기가 필요하다고 생각했던것 같아요
요 부분도 만들면서 좀 더 나은 방법을 생각해보려고 했는데 의도한만큼 효율적인 코드를 작성하진 못했던것 같습니다 ㅠㅠ

우선은 조언주신대로 key, value 로 분기처리 하는게 훨씬 깔끔한것 같아서 반영해서 수정하도록 하겠습니다!! 🙇‍♂️

Reply:
https://github.com/depromeet/toks-web/commit/6b485b2d667f8556a76d02e4d31673be38e3b770
이전에 말씀해주셨던것 처럼 숫자값으로 반환되게 변경하였습니다~!